.github/workflows/scripts/check_todos.sh

```
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -xeuo pipefail
+
+while IFS= read -r -d '' file; do
+    matches=$((cat "$file" | awk 'tolower($0) ~ /todo/ && !/no-todocheck/') || exit 1)
```

Shellcheck (0.10.0)

[error] Shells disambiguate $(( differently or not at all. For $(command substitution), add space after $( . For $((arithmetics)), fix parsing errors.

(SC1102)